app/components/PizzaMeetingButton/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@webkom/lego-bricks';
+import { ContentList } from 'app/components/ContententList';
+import Tooltip from 'app/components/Tooltip';
+import { MeetingInvitationStatus } from 'app/store/models/MeetingInvitation';
+import type { EntityId } from '@reduxjs/toolkit';
+import type { MeetingInvitationWithUser } from 'app/reducers/meetingInvitations';
+import type { ReactionsGrouped } from 'app/store/models/Reaction';
+
+type Props = {
+  meeting: {
+    reactionsGrouped?: ReactionsGrouped[];
+    invitations: EntityId[];
+    reactions?: { author: { fullName: string }; emoji: string }[];
+  };
+  meetingInvitations: MeetingInvitationWithUser[];
+};
+
+export const PizzaMettingbutton = ({ meeting, meetingInvitations }: Props) => {
+  const pizza_users: string[] =
+    meeting.reactions
+      ?.filter((reaction) => reaction.emoji === ':pizza:')
+      .map((reaction) => reaction.author.fullName)
+      .sort() ?? [];
```

Comment:
Why sort it?

Reply:
I just thought it would be easier to look for a name if the table was sorted, but I can just remove it

Reply:
hmm okay that's not so dumb - I can back that!